cfme/utils/net.py

```diff
@@ -180,6 +180,27 @@ def find_pingable(mgmt_vm, allow_ipv6=True):
         return getattr(mgmt_vm, 'ip', None)
 
 
+def find_pingable_ipv6(mgmt_vm):
```

Comment:
_Optional_ rather than having a new method, thoughts on having this be a parameter to the current `find_pingable` method? Something like `find_ipv6=False/True`

Reply:
Why does `find_pingable` not work here? 

https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/net.py#L159

Reply:
@jawatts it allows it, meaning it can return whatever it finds but in this case I need only ipv6.